helm/templates/olm/konveyor-operator.clusterserviceversion.yaml

```diff
@@ -27,6 +27,11 @@ metadata:
     operatorframework.io/suggested-namespace: konveyor-tackle
     repository: https://github.com/konveyor/tackle2-operator
     support: https://github.com/konveyor/tackle2-operator/issues
+  labels:
+    operatorframework.io/arch.amd64: supported
+    operatorframework.io/arch.arm64: supported
```

Comment:
```suggestion
    operatorframework.io/arch.arm64: unsupported
```

Reply:
@rayfordj we want arm64 supported upstream.